app/views/admin/loan_questions/_form.slim

```diff
@@ -17,20 +17,9 @@
     = f.input :data_type, as: :select, label: t('loan_questions.data_type'),
```

Comment:
This label should be pulled from simple form. Please also fix any other labels on this form.

Form labels should also be Title Case. Currently this one is 'Data type'.